src/js/components/Drop/DropContainer.js

```diff
@@ -63,7 +64,7 @@ const DropContainer = forwardRef(
       [portalContext, portalId],
     );
     const dropRef = useRef();
-    useEffect(() => {
+    useLayoutEffect(() => {
```

Comment:
I like `useLayoutEffect()` for re-calculating `place()`. But, it seems expensive to be adding and removing event listeners on each layout. Instead, I'm wondering if we should separate out `place()` via `useCallback()` and call it from the listeners as well as the layout effect instead.